installer/stretch/extensions/xdebug.sh

```diff
@@ -8,13 +8,13 @@ function install_xdebug()
                 XDEBUG_PACKAGE="xdebug-2.5.5"
                 # Build xdebug manually to avoid a debian compiler bug
                 # https://github.com/docker-library/php/issues/133
-                pushd /usr/src
+                pushd /usr/src || return 1
```

Comment:
thought we were set -e as a whole

Reply:
Shellcheck fix. Could disable the check for that or leave it be.

Reply:
would an explicit set -e in this file also work around it?

Reply:
Doesn't appear to be considered: https://github.com/koalaman/shellcheck/wiki/SC2164
But we do handle it in `enable.sh`/`disable.sh` as the first thing in the script.

Reply:
btw my rational being if code looks like it's +e when its -e, or vice versa, can be confusing and cause someone to write in the wrong +/- errexit way

Reply:
We can do both :)

Reply:
How does https://github.com/my127/docker-php/pull/28/commits/d945032320c7ba160360153f021de1d5789b054c look?

Reply:
does that solve shellcheck's complaint without return?

Reply:
Oh, yes on my local copy. Let's see what travis does

Reply:
ok, if no way to rationally be able to avoid the complaint without a disable, then I guess this is the best compromise